package/network/services/dropbear/files/dropbear.init

```diff
@@ -148,6 +148,11 @@ service_triggers()
 	procd_add_validation validate_section_dropbear
 }
 
+shutdown() {
+	# close all open connections
+	killall dropbear
```

Comment:
`killall` sounds dangerous.

How about `pkill dropbear` or `kill -TERM $(pgrep dropbear)`?

Or even better, how about a test to determine whether dropbear process is still running?
`pgrep dropbear && kill -TERM $(pgrep dropbear)`